src/components/NewsAndArticles/index.jsx

```diff
@@ -0,0 +1,180 @@
+            heading,
+            description,
+            link,
+            articles: home_articles
+          },
+          loading: false,
+          error: false
+        });
+      })
+      .catch(err => {
+        this.setState({
+          loading: false,
+          error: true,
+          errorMessage: 'An Error occurred while getting the data, '
+        });
+      });
+  };
+
+  render() {
+    // Heading content
+    const { heading_primary, heading_secondary } = this.state.data.heading;
+
+    // Link content
+    const { text, url } = this.state.data.link;
+
+    // Articles content
+    const { articles } = this.state.data;
+    //debugger;
+    const articlesList = articles ? (
+      articles.map(
+        ({
+          title,
+          link: { text, url: linkURL },
+          image: { url: imageURL },
+          _id
+        }) => (
+          <Article
+            title={title}
+            linkText={text}
+            linkURL={linkURL}
+            imageURL={imageURL}
+            key={_id}
+          />
+        )
+      )
+    ) : (
+      <div>Sorry, couldn't find the articles</div>
+    );
+
+    if (this.state.loading) {
+      return <div>loading data.....</div>;
+    } else if (this.state.error) {
+      return (
+        <div>
+          <p>
+            {this.state.errorMessage}
+            <span className="cursor-pointer text-c200" onClick={this._getData}>
+              Retry?
+            </span>
+          </p>
+        </div>
+      );
+    } else {
+      return (
+        <section className="news font-body bg-c800 mb-20 md:mb-48 pt-18 pb-1 md:pb-40 relative">
+          <div className="container">
+            <div className="head-section grid grid-cols-1 md:grid-cols-12">
+              <Heading
+                primaryText={heading_primary + ' '}
+                secondaryText={heading_secondary}
+                primaryTextColor="dark"
+                primaryClassName="md:col-span-5"
+              />
+
+              <p className="text-c600 text-lg md:mt-3 md:col-span-4 leading-relaxed">
+                {this.state.data.description}
+              </p>
+              <div className="btn-div md:col-span-3">
+                <a
+                  className="btn-sm text-sm bg-c300 my-8 md:float-right md:mt-3 cursor-pointer"
+                  href={url}
+                >
+                  {text}
+                </a>
+              </div>
+            </div>
+          </div>
+          <div className="container relative">
+            <div className="articles grid grid-cols-1 mt-12 md:mt-auto md:grid-cols-3 gap-8 md:absolute w-full">
+              {articlesList}
+            </div>
+          </div>
+        </section>
+      );
+    }
+  }
+}
+
+class Article extends Component {
+  state = {
+    isHovered: false
+  };
+
+  _setHover = () => {
+    this.setState({
+      isHovered: true
+    });
+  };
+
+  _resetHover = () => {
+    this.setState({
+      isHovered: false
+    });
+  };
```

Comment:
I think we can do the hover effect in CSS and make.
unless you have a strong to manage hover effect in the `Article  `component state.  ?